### ISSUE TITLE
Getting nextSibling element

### ISSUE THREAD

**Issue by fbzl** [Imangazaliev/DiDOM#43]
will be added this in `class Element`
```
public function nextSibling()
 {
    return $this->node->nextSibling;
 }
```

**Comment by Imangazaliev** [comment]
Ok

**Comment by fbzl** [comment]
Tanks!

**Comment by Imangazaliev** [comment]
Released! See [1.7.0](https://github.com/Imangazaliev/DiDOM/releases/tag/1.7.0)
